packages/kbn-unified-data-table/src/components/data_table.scss

```diff
@@ -101,6 +93,20 @@
   }
 }
 
+// Custom styles for data grid header cell.
+// It can also be inside a portal (outside of `unifiedDataTable__inner`) when dragged.
+.unifiedDataTable__headerCell {
+  align-items: start;
+
+  .euiDataGridHeaderCell__draggableIcon {
+    padding-block: 2px // to align with a token height
```

Comment:
nit :)
This could be...
```
padding-block: $euiSizeXS / 2;
```

Reply:
Updated via https://github.com/elastic/kibana/pull/197832/commits/7967c37883e17b26ebcb65a0abb7d8fecfb269f5, thanks!